src/components/model/ModelAssociationsListItem.svelte

```diff
@@ -13,8 +14,7 @@
   import { permissionHandler } from '../../utilities/permissionHandler';
   import { tooltip } from '../../utilities/tooltip';
   import Collapse from '../Collapse.svelte';
-  import ContextMenu from '../context-menu/ContextMenu.svelte';
-  import ContextMenuItem from '../context-menu/ContextMenuItem.svelte';
+  import ContextMenuInternal from '../context-menu/ContextMenu.svelte';
```

Comment:
I think line 62 also needs to be renamed to use the `ContextMenuInternal` type?

Reply:
Good catch, done